src/app/query/stacks/balance/account-balance.utils.ts

```diff
@@ -23,7 +23,10 @@ export function createStxCryptoAssetBalance(
 
   return {
     availableBalance: subtractMoney(totalBalance, outboundBalance),
-    availableUnlockedBalance: subtractMoney(unlockedBalance, outboundBalance),
+    // Temporary fix for negative balances returning from the API
+    availableUnlockedBalance: unlockedBalance.amount.isPositive()
+      ? subtractMoney(unlockedBalance, outboundBalance)
+      : createMoney(0, 'STX'),
```

Comment:
Btw on this I wonder if building out some math utils for money would be better,

e.g.

`Money.max(createMoney(0, 'STX'), unlockedBalance.amount)`

Reply:
Is this what your thinking? https://github.com/leather-io/mono/pull/1273